plugins/modules/iosxr_bgp_neighbor_address_family.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2021 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for iosxr_bgp_neighbor_address_family
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: iosxr_bgp_neighbor_address_family
+short_description: Manages BGP neighbor address family resource module.
+description:
+- This module configures and manages the attributes of BGP global on Cisco IOS-XR platforms.
+version_added: 2.0.0
+author: Ashwini Mhatre (@amhatre)
+notes:
+- Tested against Cisco IOS-XR 6.1.3.
+- This module works with connection C(network_cli).
+options:
+    config:
+      description: A list of configurations for BGP address family.
```

Comment:
`config` is a dict and not list.
```suggestion
      description: BGP neighbor address family configurations.
```

Reply:
Ack,made changes